modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,7 +33,14 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
 
 You can specify the version of the proxy you want to use by clicking the *Advanced Options* drop-down menu and selecting the proxy version that best covers your needs.
 
-See xref:proxy-deploy-raml-oas-proxy.adoc[Building a RAML/OAS API Proxy] for more information.
+== API Proxy and Validations
```

Comment:
Please consider including:
- That the proxy validations are available for mule 3 and 4.
- That the wsdl and raml proxies have this new feature (Seems that these docs only talk about the raml/rest one)
- Which are the defaults of the validation types, depending on the proxy type (wsdl/raml/rest) and the mule version (3x/4)
- Consider talking that for mule 3 runtimes, these feature is supported for 3.8.x and onwards.
- By enabling the validation:
in the raml proxy: the payload (against a schema), query parameters, uri parameters, headers and form parameters  are validated (There may be an apikit document talking about these validations, consider linking it if it exists).
in the wsdl proxy, the envelope is validated against the structure provided in the wsdl file.
- It should be clear that raml/rest and wsdl proxies will have the validation flags feature, but strict validation is only for mule 4 raml/rest proxy

Reply:
Done! Thanks!